### ISSUE TITLE
[[Warning]] : Converting a tensor with requires_grad=True to a scalar may lead to unexpected behavior

### ISSUE THREAD

**Issue by miroi** [CederGroupHub/chgnet#245]
### Email (Optional)

miro.ilias@gmail.com

### Version

v0.4.2

### Which OS(es) are you using?

- [ ] MacOS
- [ ] Windows
- [x] Linux

### What happened?

Hello,

with up-to-date pip installed CHGNet and on my WSL2 Linux I tried simple md test from https://github.com/CederGroupHub/chgnet/tree/e2a2b82bf2c64e5a3d39cd75d0addfa864a2771a?tab=readme-ov-file#molecular-dynamics 

would you please help  ? (Torch version 2.9.0)

~~~
python md_test.py
CHGNet v0.3.0 initialized with 412,525 parameters
CHGNet will run on cuda
CHGNet will run on cuda
NVT-Berendsen-MD created
/home/miroi/work/software/myenv/lib/python3.12/site-packages/chgnet/model/model.py:898: UserWarning: Converting a tensor with requires_grad=True to a scalar may lead to unexpected behavior.
Consider using tensor.detach() first. (Triggered internally at /pytorch/torch/csrc/autograd/generated/python_variable_methods.cpp:836.)
  volumes = torch.tensor(volumes, dtype=TORCH_DTYPE, device=atomic_numbers.device)
~~~

### Code snippet

```python

```

### Log output

```shell

```

### Code of Conduct

- [x] I agree to follow this project's Code of Conduct

**Comment by miroi** [comment]
PS: This is not bug, rather warning, to avoid bug in practical computations....
